x/posts/module.go

```diff
@@ -213,13 +200,8 @@ func (AppModule) GenerateGenesisState(simState *module.SimulationState) {
 	simulation.RandomizeGenState(simState)
 }
 
-// ProposalContents doesn't return any content functions for governance proposals.
-func (AppModule) ProposalContents(_ module.SimulationState) []simtypes.WeightedProposalContent {
```

Comment:
Will open a new PR to remove `x/params` and implements all `ProposalMsgs` methods for modules.
https://github.com/cosmos/cosmos-sdk/blob/release/v0.47.x/UPGRADING.md#simulation